lua/plugins/filetype.lua

```diff
@@ -154,16 +154,22 @@ vis.ftdetect.filetypes = {
 	gap = {
 		ext = { "%.g$", "%.gd$", "%.gi$", "%.gap$" },
 	},
-	gemini = {
-		ext = { "%.gmi" },
-		mime = { "text/gemini" },
-	},
+--  Gemini lexer is truly not supported anymore, if anybody cares about it
+--  uncomment this field
+--	gemini = {
+--		ext = { "%.gmi" },
+--		mime = { "text/gemini" },
+--	},
```

Comment:
Probably better to just remove it - people can still add it in their own rc-file it they want this.

Reply:
Actually, it has been in the end merged into Scintillua https://github.com/orbitalquark/scintillua/pull/74.